CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+### v2.0.0-beta4
+
+#### Features
+
+ - [`e9d102b`](https://github.com/deis/monitor/commit/e9d102b06667efadb43c34d9e1b6dd08e5ee6efd) grafana: Use grafana to calc request per second
+ - [`5ce1dbf`](https://github.com/deis/monitor/commit/5ce1dbf3484eb3a43b60f6f8c3cdd62b56f06fe3) telegraf: Add the ability to poll kubernetes prometheus endpoints
+
+#### Fixes
+
+ - [`7a156c4`](https://github.com/deis/monitor/commit/7a156c47e50ecdcad1af2068a5bab34720a0bc70) telegraf: Set insecure verify to true
+ - [`6b2f8bb`](https://github.com/deis/monitor/commit/6b2f8bbba593412cf0811c46c952cd4465df95df) telegraf: dont remove curl
+ - [`2fe7927`](https://github.com/deis/monitor/commit/2fe79274650d141febce12dcf3fbfabdb1b0e600) telegraf: Document agent configuration
+ - [`8ea1bd7`](https://github.com/deis/monitor/commit/8ea1bd79c96ee6d7491baa2aecb11e335a454406) influx: Make influx config a go template
+
+#### Documentation
```

Comment:
can you remove this section or write "No documentation changes this release" in this section